android/sdk/src/main/java/io/o2mc/sdk/Datastreams/Connectivity.java

```diff
@@ -15,58 +15,76 @@
  * Class pulled from Github. (With some small changes)
  * It returns the current networkstate as a String
  */
+@SuppressWarnings("WeakerAccess")
 public class Connectivity {
 
     /**
      * Get the network info
+     *
      * @param context
-     * @return
+     * @return null if unable to retrieve active network information
      */
-    public static NetworkInfo getNetworkInfo(Context context){
+    @SuppressWarnings("JavaDoc")
```

Comment:
not sure why we would want to suppress this javadoc warning

Reply:
the concept of the 'context' variable is clear to any android developer, it's like explaining 'what does "hello" mean' metaphorically speaking
the "weakeraccess" warning is one that's generally ignored for good reasons: when you're working on a project which is in active development, you're usually not entirely sure when you'd call which class and from where. the Connectivity class is not used a lot yet, it's a remnant from old code. it may or may not become useful later on, which is the reason why it's still here in the first place and why it should stay public (for now).